app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -64,46 +64,27 @@ public function store(Request $request, $seatId = null, $backTo = null)
 
         $this->authorize('checkout', $license);
 
-        if (! $license->reassignable) {
+        if (!$license->reassignable) { // FIXME/TODO - should _this_ go into the checkin method? YES! No?
             // Not allowed to checkin
             Session::flash('error', trans('admin/licenses/message.checkin.not_reassignable') . '.');
 
             return redirect()->back()->withInput();
         }
 
-        // Declare the rules for the form validation
-        $rules = [
-            'notes'   => 'string|nullable',
-        ];
-
-        // Create a new validator instance from our validation rules
-        $validator = Validator::make($request->all(), $rules);
-
-        // If validation fails, we'll exit the operation now.
-        if ($validator->fails()) {
-            // Ooops.. something went wrong
-            return redirect()->back()->withInput()->withErrors($validator);
-        }
-
         if($licenseSeat->assigned_to != null){
-            $return_to = User::find($licenseSeat->assigned_to);
```

Comment:
I think `$return_to` here ties into the redirector that @Godmartinz wrote. Not sure we can pull it from here, but we'll need to test further.